README.Rmd

```diff
@@ -80,147 +103,54 @@ remotes::install_github("epiforecasts/EpiNow2")
+<details> <summary> Getting Started </summary>
 
-```{r, message = FALSE, warning = FALSE}
-estimates <- epinow(
-  reported_cases = reported_cases,
-  generation_time = generation_time_opts(example_generation_time),
-  delays = delay_opts(example_incubation_period + reporting_delay),
-  rt = rt_opts(prior = list(mean = 2, sd = 0.2)),
-  stan = stan_opts(cores = 4, control = list(adapt_delta = 0.99)),
-  verbose = interactive()
-)
-names(estimates)
-```
+The [Getting Started vignette](https://epiforecasts.io/EpiNow2/dev/EpiNow2.html)
+is your quickest entry point to the package. It provides a quick run through of
+the two main functions in the package and how to set up them up. It also
+discusses how to summarise and visualise the results after running the models.
 
-Both summary measures and posterior samples are returned for all parameters in an easily explored format which can be accessed using `summary`. The default is to return a summary table of estimates for key parameters at the latest date partially supported by data. 
+</details>
 
-```{r}
-knitr::kable(summary(estimates))
-```
+<details> <summary> Package website </summary>
 
-Summarised parameter estimates can also easily be returned, either filtered for a single parameter or for all parameters.
+The [package website](https://epiforecasts.io/EpiNow2/dev/index.html) provides
+various resources for learning about the package, including the function
+reference, details about each model (model definition), workflows for
+each model (usage), and case studies or literature of applications of
+the package.
 
-```{r}
-head(summary(estimates, type = "parameters", params = "R"))
-```
+</details>
 
-Reported cases are returned in a separate data frame in order to streamline the reporting of forecasts and for model evaluation.
+<details> <summary> End-to-end workflows </summary>
 
-```{r}
-head(summary(estimates, output = "estimated_reported_cases"))
-```
-
-A range of plots are returned (with the single summary plot shown below). These plots can also be generated using the following `plot` method.
+The [workflow vignette](https://epiforecasts.io/EpiNow2/dev/articles/estimate_infections_workflow.html) provides guidance on the end-to-end process of estimating reproduction numbers and performing short-term forecasts for a disease spreading in a given setting.
 
-```{r, dpi = 330, fig.width = 12, fig.height = 12, message = FALSE, warning = FALSE}
-plot(estimates)
-```
+</details>
 
+<details> <summary> Model definitions </summary>
 
-### [regional_epinow()](https://epiforecasts.io/EpiNow2/reference/regional_epinow.html)
+On the website, we provide the mathematical definition of each model.
+For example, the model definition vignette for `estimate_infections()` can be found [here](https://epiforecasts.io/EpiNow2/dev/articles/estimate_infections.html).
```

Comment:
As mentioned above we have a bit of an issue that we have multiple versions of the website. Here we sometimes link to the dev version (for vignettes) and sometimes the stable version (for references). Ideally these would be relative links so that each README links to the "correct" version.

Reply:
As above need to chase up CRAN policy on this so we don't get shouted at

Reply:
I couldn't find a related CRAN policy and have been digging around for examples of CRAN packages that use relative links but haven't found any. A good candidate would have been [readr](https://readr.tidyverse.org/dev/) but I don't see them using relative links.